app/controllers/application_controller.rb

```diff
@@ -36,7 +36,8 @@ class ApplicationController < ActionController::Base
   helper_method :current_templates
   helper_method :admin?, :logged_in?, :highlight_all
 
-  #  protect_from_forgery :secret => '112141be0ba20082c17b05c78c63f357'
+  protect_from_forgery with: :exception
```

Comment:
I don't think forgery protection is enabled in tests.


__

Vikrant Chaudhary
http://webstream.io


On 2 March 2016 at 20:44, Zaid Akram <notifications@github.com> wrote:

> In app/controllers/application_controller.rb
> <https://github.com/ari/jobsworth/pull/541#discussion_r54735999>:
>
> > @@ -36,7 +36,8 @@ class ApplicationController < ActionController::Base
> >    helper_method :current_templates
> >    helper_method :admin?, :logged_in?, :highlight_all
> >
> > -  #  protect_from_forgery :secret => '112141be0ba20082c17b05c78c63f357'
> > +  protect_from_forgery with: :exception
>
> Are you referring to EmailsController ? Looking at its tests and test.logs
> after running tests, I think its working fine. But let me still revisit it,
> just to be sure... Specially those controller which don't have proper test
> coverage.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ari/jobsworth/pull/541/files#r54735999>.
>